liquibase-standard/src/main/java/liquibase/datatype/core/CharType.java

```diff
@@ -1,18 +1,22 @@
 package liquibase.datatype.core;
 
+import liquibase.Scope;
 import liquibase.change.core.LoadDataChange;
 import liquibase.database.Database;
 import liquibase.database.core.H2Database;
 import liquibase.database.core.MSSQLDatabase;
+import liquibase.database.core.OracleDatabase;
 import liquibase.database.core.PostgresDatabase;
 import liquibase.datatype.DataTypeInfo;
 import liquibase.datatype.DatabaseDataType;
 import liquibase.datatype.LiquibaseDataType;
+import liquibase.integration.commandline.LiquibaseCommandLineConfiguration;
 import liquibase.statement.DatabaseFunction;
 import liquibase.util.StringUtil;
 
 import java.math.BigInteger;
 import java.util.Arrays;
+import java.util.List;
 import java.util.Locale;
 
 @DataTypeInfo(name="char", aliases = {"java.sql.Types.CHAR", "bpchar", "character"}, minParameters = 0, maxParameters = 1, priority = LiquibaseDataType.PRIORITY_DEFAULT)
```

Comment:
The issue mentions CLOB, but the change is being done to char (and varchar by extension). Should it also be applied to clob or the description could be changed?

Reply:
Liquibase treats it like a varchar insert, not a clob, because the Java data type is a string. This line is where Liquibase infers the type to be a CharType, despite the insert going into a clob column: https://github.com/liquibase/liquibase/blob/master/liquibase-standard/src/main/java/liquibase/sqlgenerator/core/InsertGenerator.java#L73